Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -294,7 +295,8 @@ open class MobileSharedApplication: SharedApplication {
     let sharedApplication: UIApplication
     let controller: UIViewController?
     let openURL: ((URL) -> Void)
-
+    
+    private var authChannel: AnyObject?
```

Comment:
need to retain the sessions or they will not appear

Reply:
Can you give this a proper type now that the iOS 11 code is gone?

Reply:
I have it holding the SFSafariViewController now, but I'll revert that so it's the original code and make authChannel the class type

Reply:
renamed to your suggestion over slack and added comment